strawberryfields/parameters.py

```diff
@@ -350,6 +350,13 @@ def _eval_evalf(self, prec):
                     self
                 )
             )
+
+        # remove unnecessary dims when returning measurement
+        try:
+            res = np.squeeze(res).item()
+        except ValueError:
+            res = np.squeeze(res)
```

Comment:
This feels a bit inelegant/hacky to me. Both the `try/except` and the fact that we need to do the squeezing in this part of the stack. It signals to me that we still don't have a rock-solid "unified" contract between the backends and frontend